roles/ceph-docker-common/tasks/misc/ntp_suse.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: include ../checks/check_ntp_suse.yml
```

Comment:
Where is  `check_ntp_suse.yml`?
Also, can you merge this with `ntp_redhat.yml`?